src/scenes/Languages/List/LanguageList.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Grid, makeStyles, Typography } from '@material-ui/core';
+import { times } from 'lodash';
+import React, { FC } from 'react';
+import { Language } from '../../../api';
+import { LanguageListItemCard } from '../../../components/LanguageListItemCard';
+import { SortButtonDialog, useSort } from '../../../components/Sort';
+import { useLanguagesQuery } from './languages.generated';
+import { LanguageSortOptions } from './LanguageSortOptions';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+  },
+  options: {
+    margin: spacing(3, 0),
+  },
+}));
+
+export const LanguageList: FC = () => {
+  const sort = useSort<Language>();
+
+  const { loading, data } = useLanguagesQuery({
+    variables: {
+      input: {
+        ...sort.value,
+      },
+    },
+  });
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Typography variant="h2" paragraph>
+        My Languages
```

Comment:
Technically I don't think this is "my" languages. Project list needs the "my" filter, but it isn't implemented by the API yet. Let's leave it off for now.

```suggestion
        Languages
```